addon/styles/components/_s-tag.scss

```diff
@@ -11,3 +11,28 @@
     width: 12px;
   }
 }
+
+&.important{
+  background: $q-sky !important;
+  color: $themePrimaryGrey !important;
```

Comment:
I think we don't need **!important** if we are going with higher specificity approach